src/repo.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * All repo relate configuration lives within this file. Add domains,
+ * effects, and configure any repo startup behavior here.
+ */
+
+import Microcosm from 'microcosm'
+import Game from './domains/game'
+import Timer from './effects/timer'
+
+export default class Repo extends Microcosm {
```

Comment:
Repo is the canonical name for a Microcosm instance. It's short for `repository`.